src/Application/Query/Auth/GetToken/GetTokenHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Query\Auth\GetToken;
+
+use App\Application\Query\QueryHandlerInterface;
+use App\Domain\User\Auth\AuthenticationProviderInterface;
+use App\Domain\User\Query\Repository\UserReadModelRepositoryInterface;
+
+class GetTokenHandler implements QueryHandlerInterface
+{
+    public function __invoke(GetTokenQuery $query)
```

Comment:
Just one question, do you order the methods by the visibility? First public then protected and then private?

Reply:
By purpose but it use to match with visibility.